lutris/runners/wine.py

```diff
@@ -411,6 +431,20 @@ class wine(Runner):
             "default": lambda: dgvoodoo2Manager().version,
             "conditional_on": "dgvoodoo2",
         },
+        {
+            "option": "proton_hdr",
+            "section": _("Graphics"),
+            "label": _("Enable HDR (Experimental)"),
+            "type": "bool",
+            "default": False,
+            "advanced": True,
+            "visible": _get_wine_wayland_warning,
+            "conditional_on": "Graphics",
```

Comment:
This won't work- "conditional_on" works for boolean settings, but "Graphics" is not a boolean.

We need to write a fancy "conditional" here- it needs to check if Wayland is selected *and* that the Wine version is Proton.

Reply:
I guess we can just drop the `conditional_on`, because `visible` should be enough already?

Reply:
Well, make sure "visible" is correct- now that you mention it, it looks off. A warning were a boolean is expected? I think that'll do the opposite of what you'd want- visible if there is a warning, invisible if not.

Reply:
> Well, make sure "visible" is correct- now that you mention it, it looks off. A warning were a boolean is expected? I think that'll do the opposite of what you'd want- visible if there is a warning, invisible if not.

Sorry, yes in the previous version I used same function as to get warning, but current version should be better I think since I returned your `_is_proton_wayland_available` function from https://github.com/lutris/lutris/pull/6127  in a simplified form for that.